.github/workflows/update-rpm-repo.yml

```
@@ -0,0 +1,279 @@
+              git add -A
+
+              # Check if there are changes to commit
+              if git diff --staged --quiet; then
+                echo "No changes to commit"
+                return 0
+              fi
+
+              # Commit
+              git commit -m "$(cat <<'EOF'
+feat: add Node.js ${VERSION} package to RPM repository
+
```

actionlint (1.7.8)

could not parse as YAML: did not find expected key

(syntax-check)